packages/cron/src/lib/utils.js

```diff
@@ -102,13 +123,30 @@ function getPgConnString (env, mode = 'rw') {
  * @param {Record<string, string|undefined>} env
  */
 export function getCargoPgPool (env) {
-  const connection = env.CARGO_PG_CONNECTION
-  if (!connection) {
-    throw new Error('Missing CARGO_PG_CONNECTION string. Please add it to the environment.')
-  }
+  const cargoEnvVariables = [
+    'DAG_CARGO_HOST',
+    'DAG_CARGO_DATABASE',
+    'DAG_CARGO_USER',
+    'DAG_CARGO_PASSWORD'
+  ]
+
+  cargoEnvVariables.forEach((variable) => {
+    if (!env[variable]) {
+      throw new Error(`Missing ${variable} string. Please add it to the environment.`)
+    }
+  })
+
+  const connectionString = buildPgConnectionString({
+    user: env.DAG_CARGO_USER,
+    database: env.DAG_CARGO_DATABASE,
+    password: env.DAG_CARGO_PASSWORD,
+    host: env.DAG_CARGO_HOST
+  })
+
+  console.log(`Using ${connectionString}`)
```

Comment:
Is there a danger of this logging the DB password somewhere? Or is `console.log` only ever hooked up to something in a dev/test environment?